test/build.info

```diff
@@ -260,7 +260,7 @@ IF[{- !$disabled{tests} -}]
 
   INCLUDE[testutil.o]=..
   INCLUDE[ssl_test_ctx.o]=../include
-  INCLUDE[handshake_helper.o]=../include
+  INCLUDE[handshake_helper.o]=../include ../ssl
```

Comment:
Unless the intended program is meant to test internals, this isn't a good idea.  Does this even link correctly with shared libraries?